Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN apk --no-cache add git linux-headers ca-certificates
 COPY . /go/src/github.com/ChainSafeSystems/go-pre
 WORKDIR /go/src/github.com/ChainSafeSystems/go-pre
 ENV GO111MODULE=off
-RUN go mod vendor
+RUN go install
```

Comment:
This will indicate no `no Go files in dir`, as they're no go files in root. Go install will pull in dependencies when running within packages such as `go install ./...`. See go-ethereum/build/ci.go function name `doInstall` as an example. 

Should be `go install -v ./...`